timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -422,7 +422,11 @@ module.exports = [
         href: "update-timescaledb",
         children: [
           {
-            title: "Update TimescaleDB from 1.x to 2.x",
+            title: "Update TimescaleDB",
+            href: "update-timescaledb"
+          },
+          {
+            title: "Update from TimescaleDB 1.x to 2.x",
```

Comment:
I don't like this now (or I missed it earlier). With this set up we end up having two nodes that both say "update TimescaleDB" but offer different content. How do we fix this?

Ideas @Loquacity ??

Reply:
I guess you could use this construction: "Upgrade to [product] [version]". Most people don't really think about what they're upgrading *from*, they just want the next version. Does that help?

Reply:
Right, I think I wasn't clear (no surprise!)

 * The "parent" is currently the landing page and the content comes from "index.md" in that folder.
 * The child "Update TimescaleDB" is really "updating within any major version" (not from 1.x to 2.x). Its content is in `update-timescaledb.md`
 * The child "update from timescaledb 1.x to 2.x" was created back in November to give people some extra context on changes between major versions. Updating the extension was still a very quick (one SQL command) process, but we wanted to provide more background, etc.

So, the two child pages are the same when talking about _how_ to update the extension (that doesn't change), the difference was that we made breaking changes and felt like we needed to give more context. Maybe we don't know, and we just put the 1.x to 2.x stuff as a section at the bottom and say at the top, "if you're updating from 1.x to 2.x, please see this page first for more info..."??

Reply:
Didn't have time to do this today, but would you be OK with this approach:

Remove `update-timescaledb.md` (my new file) and just integrate those contents into the `index.md` file. That way clicking on "Update TimescaleDB" just gets right to the "how do I update" process. We'd keep the note at the top about seeing the 1.x to 2.x.

I feel like this simplifies it for now and doesn't end up with a URL like `....\update-timescaledb\update-timescaledb` which is just silly.

And yes, this all needs to be reorganized, but I'd like to get this update out sooner than later since a few community members have recently asked the simple question of "how do I update TimescaleDB 2.x?"

Reply:
Ah, I understand. Yep, let's do that for now, and we can re-address when we implement the IA changes later on 👍